src/rafimage.cpp

```diff
@@ -296,6 +292,37 @@ namespace Exiv2 {
         io_->read(jpg_img_length, 4);
         long jpg_img_off = Exiv2::getULong((const byte *) jpg_img_offset, bigEndian);
         long jpg_img_len = Exiv2::getULong((const byte *) jpg_img_length, bigEndian);
+        byte cfa_header_offset [4];
+        io_->read(cfa_header_offset, 4);
```

Comment:
Before doing all these reads, please check that the image is actually large enough, or check the return value of each `io_->read()`.